x-pack/plugins/security_solution/public/resolver/view/use_camera.test.tsx

```diff
@@ -247,25 +247,24 @@ describe('useCamera on an unpainted element', () => {
       expect(simulator.mock.requestAnimationFrame).not.toHaveBeenCalled();
     });
     describe('when the camera begins animation', () => {
-      let process: SafeResolverEvent;
+      let node: ResolverNode;
       beforeEach(async () => {
-        const events: SafeResolverEvent[] = [];
-        const numberOfEvents: number = 10;
+        const nodes: ResolverNode[] = [];
+        const numberOfNodes: number = 10;
 
-        for (let index = 0; index < numberOfEvents; index++) {
-          const uniquePpid = index === 0 ? undefined : index - 1;
-          events.push(
-            mockProcessEvent({
-              endgame: {
-                unique_pid: index,
-                unique_ppid: uniquePpid,
-                event_type_full: 'process_event',
-                event_subtype_full: 'creation_event',
-              },
+        for (let index = 0; index < numberOfNodes; index++) {
+          const parentID = index === 0 ? undefined : String(index - 1);
+          nodes.push(
+            mockResolverNode({
+              id: String(index),
+              name: '',
+              parentID,
+              timestamp: 1582233383000,
```

Comment:
What is the significance of this number? If there is none, can you add a comment and/or make it `0` for consistency w/ the rest of this test (which uses null-ish values like 0 or '' when a value isn't important.)

Reply:
came from mockProcessEvent to maintain parity there, but doesn't look like it's necessary. will update it